src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -361,7 +361,7 @@ protected function describeContainerDefinition(Definition $definition, array $op
         }
 
         $inEdges = null !== $builder && isset($options['id']) ? $this->getServiceEdges($builder, $options['id']) : [];
-        $tableRows[] = ['Usages', $inEdges ? implode(', ', $inEdges) : 'none'];
+        $tableRows[] = ['Usages', $inEdges ? implode(\PHP_EOL, $inEdges) : 'none'];
```

Comment:
what about limiting to eg 3 and show "and  more" instead? I agree with you that the long out put is useless

Reply:
@nicolas-grekas was thinking about it, but isn't it the main reason of debug command to clearly show the details about the resource?
Another option I thought was to implement a new command argument which would show usages (similar to `show-arguments`), but this would result to BC break, because by default it would no longer show usages..
IMO leaving all usages, but showing them in a single column is better (attaching screenshots of differences)
WDYT?

![image](https://user-images.githubusercontent.com/14221532/228779322-79359200-d894-4218-b963-9fef400b71fd.png)
![image](https://user-images.githubusercontent.com/14221532/228779436-8a480396-769a-4d2f-a5fb-e4371bbf8696.png)